src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpWindowsProxy.cs

```diff
@@ -2,21 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO.Compression;
 using System.Net.NetworkInformation;
 using System.Runtime.InteropServices;
 using System.Text.RegularExpressions;
-
+using System.Threading;
 using SafeWinHttpHandle = Interop.WinHttp.SafeWinHttpHandle;
 
 namespace System.Net.Http
 {
     internal sealed class HttpWindowsProxy : IWebProxy, IDisposable
     {
-        private readonly Uri _insecureProxyUri;         // URI of the http system proxy if set
-        private readonly Uri _secureProxyUri;         // URI of the https system proxy if set
-        private readonly List<Regex> _bypass;           // list of domains not to proxy
-        private readonly bool _bypassLocal = false;     // we should bypass domain considered local
+        const int CacheCleanupFrequencyMilliseconds = 30 * 1000;
```

Comment:
These 'const' fields need the 'private' access modifier.  We typically also put 'const' items above the other fields.